src/ray/raylet/local_object_manager.cc

```diff
@@ -74,7 +75,8 @@ void LocalObjectManager::WaitForObjectFree(const rpc::Address &owner_address,
 
     core_worker_subscriber_->Subscribe(
         std::move(sub_message), rpc::ChannelType::WORKER_OBJECT_EVICTION, owner_address,
-        object_id.Binary(), subscription_callback, owner_dead_callback);
+        object_id.Binary(), /*subscribe_done_callback=*/{}, subscription_callback,
```

Comment:
For most places in ray we use nullptr for no callback, do you want to follow this pattern?